projects/adf5902_sdz/src/app_iio.c

```diff
@@ -0,0 +1,122 @@
+/***************************************************************************//**
```

Comment:
You can use iio/iio_app (see iio_demo).
Otherwise we will duplicate the same iio code in every project

Reply:
Is there any actual no-OS project that uses this approach (not demo projects)? Otherwise I would change (and test) all no-OS projects with the suggested improvement in a separate PR/commit .

Reply:
https://github.com/analogdevicesinc/no-OS/blob/master/projects/ad7124-8pmdz
When iio_app was added only iio_demo was modified as example for new projects.
The others projects where not modified.